server/app/views/api/ApiDocsView.java

```diff
@@ -164,7 +155,10 @@ private DivTag programDocsDiv(ProgramDefinition programDefinition) {
         div().withClasses("flex", "flex-col", "border", "border-gray-300", "rounded-lg", "m-4");
 
     for (QuestionDefinition questionDefinition :
-        programDefinition.streamQuestionDefinitions().collect(toImmutableList())) {
+        programDefinition
+            .streamQuestionDefinitions()
+            .sorted(Comparator.comparing(QuestionDefinition::getQuestionNameKey))
+            .collect(toImmutableList())) {
       programDocsDiv.with(questionDocsDiv(questionDefinition));
```

Comment:
We sort the program list (left side of API docs) so that they match the sorted JSON blob.